dnf-testing.py

```diff
@@ -1,13 +1,12 @@
+#!/usr/bin/env python
```

Comment:
Why have the refactors as a separate commit in the PR and not just squash them?

Reply:
Yeah, that could be cleaned a bit...

Reply:
Done.